functests/performance/updating_profile.go

```diff
@@ -113,12 +118,55 @@ var _ = Describe("[rfe_id:28761] Updating parameters in performance profile", fu
 			}
 		}
 	})
+
+	It("[test_id:28440]Verifies that nodeSelector can be updated in performance profile", func() {
+		var newWorkerNode corev1.Node
+		newRole := "worker-test"
+		newLabel := fmt.Sprintf("%s/%s", testutils.LabelRole, newRole)
+		newNodeSelector := map[string]string{newLabel: ""}
+
+		By("Skipping test if cluster does not have another available worker node")
+		workerNodes, err := nodes.GetByRole(testclient.Client, "worker")
+		Expect(err).ToNot(HaveOccurred())
+
+		for i := range workerNodes {
+			if _, lbl := workerNodes[i].Labels[fmt.Sprintf("%s/%s", testutils.LabelRole, testutils.RoleWorkerRT)]; lbl {
```

Comment:
AFAIk it's best practice to just use `ok` for the boolean.
Something `exists` would work as well for me though.
But `lbl` does not indicate it's a boolean, so please change this.

Reply:
yeah, you are right! thanks!